vale/Vocab/CockroachDB/accept.txt

```diff
@@ -277,6 +277,7 @@ preloaded
 preparable
 prepend(ed|ing|s)
 Prisma
+procedural[s]
```

Comment:
My brain has a case of the Mondays. I definitely made a bad suggestion here. Sorry! Use this instead:

```suggestion
procedurals?
```